src/MealCard/MealCard.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import moment from "moment";
+import "./MealCard.less";
+
+function formatPrepTimeHours(prepTime) {
+	let hours = Math.floor(parseInt(prepTime) / 60);
+
+	if(isNaN(hours)) {
+		return "";
+	}
+	else if(hours === 0) {
+		return "";
+	} else if(hours === 1) {
+		return `${hours} hour`;
+	} else {
+		return `${hours} hours`;
+	}
+}
+
+function formatPrepTimeMinutes(prepTime) {
+	let minutes = parseInt(prepTime) % 60;
+
+	if(isNaN(minutes)) {
+		return "";
+	} else if(minutes === 0) {
+		return "";
+	} else if(minutes === 1) {
+		return `${minutes} minute`;
+	} else {
+		return `${minutes} minutes`;
+	}
+}
+
+function formatPrepTime(prepTime) {
+	return `${formatPrepTimeHours(prepTime)} ${formatPrepTimeMinutes(prepTime)}`;
+}
+
+function formatLastUsed(lastUsed) {
+	const date = moment(lastUsed);
+
+	if(lastUsed && date.isValid()) {
+		return date.format("MMM Do, YYYY");
+	} else {
+		return "never used";
+	}
+}
```

Comment:
There's also no reason formatters like these couldn't be implemented in a separate package, where they could be tested in isolation and shared across components that need to display data in consistent ways.

Reply:
Yeah - I look forward to us building out a nice little library of reusable functions like this that are very generic, fully unit test covered, and reused throughout our apps.